pkgs/development/python-modules/nexusformat/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # build-system
+  setuptools,
+  setuptools-scm,
+  wheel,
+
+  # tests
+  pytest,
+
+  # dependencies
+  h5py,
+  hdf5plugin,
+  numpy,
+  scipy,
+}:
+
+buildPythonPackage rec {
+  pname = "nexusformat";
+  version = "1.0.6";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-UxU3PA/2r/uamdysbfC0L2JinHgfkXhssHIo2hf3zlA=";
+  };
+
+  pyproject = true;
+
+  build-system = [
+    setuptools
+    setuptools-scm
+    wheel
+  ];
+
+  dependencies = [
+    h5py
+    hdf5plugin
+    numpy
+    scipy
+  ];
+
+  nativeCheckInputs = [ pytest ];
+
+  checkPhase = ''
+    pytest
+  '';
```

```suggestion
  nativeCheckInputs = [ pytestCheckHook ];
```
Could you please use `pytestCheckHook` instead?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook-using-pytestcheckhook